test/__snapshots__/github.test.ts.snap

```diff
@@ -570,15 +573,80 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Setup GIT identity
-        run: git config --global user.name "github-actions" && git config --global user.email "github-actions@github.com"
+      - run: git config --global user.name "github-actions" && git config --global user.email "github-actions@github.com"
+      - name: AWS Credentials
+        uses: aws-actions/configure-aws-credentials@master
+        with:
+          role-to-assume: publishRole
+          role-session-name: GitHubAction
+          aws-region: us-east-1
+      - name: Download Artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: cloud-assembly
+          path: cdk.out/
+      - run: npx projen install:ci
+      - run: npx projen publish:assets
+"
+`;
+
+exports[`Github snapshot with independent stage 1`] = `
+"# ~~ Generated by projen. To modify, edit .projenrc.js and run "npx projen".
+
+name: deploy
+on:
+  push:
```

Comment:
see here

Reply:
That is the deploy.yaml of the normal pipeline. It runs on push and does a synth and assetUpload. No stages here as none are configured